src/ChipInput.js

```diff
@@ -576,7 +576,7 @@ class ChipInput extends React.Component {
           [classes.marginDense]: other.margin === 'dense'
         })}
         error={error}
-        required={required}
+        required={chips.length ? undefined : required}
```

Comment:
This could be changed to `chips.length === 0` so that we know what this is intended to do. Let the minifier minify the code.